modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -3,46 +3,48 @@
+By default, Redpanda enables topic-aware leadership balancing with the xref:reference:cluster-properties.adoc#enable_leader_balancer[`enable_leader_balancer`] property. Automatic partition leadership balancing improves cluster performance by transferring partition leadership from one broker to other replicas. This transfer changes where data is read from and written to first, but leadership transfer does not involve any data transfer.
 
 The xref:reference:cluster-properties.adoc#leader_balancer_mode[`leader_balancer_mode`] property ensures that each shard in a cluster has an equal number of partitions. It determines the movement of leadership for the replica set of a partition. It supports two modes:
 
-* `random_hill_climbing`: This mode randomly searches for potential leadership movements. If one is found that better balances the number of leaders per shard and the leaders of a given topic per broker, then the movement is applied to the cluster. This is the default.
-* `greedy_balanced_shards`: This mode uses a heuristic to search for leadership movements that better balance leaders per shard. It applies any movement it finds.
+* `random_hill_climbing`: This mode randomly searches for potential leadership movements. If it identifies a movement that improves the balance of leaders per shard and the leaders of a given topic per broker, then the change is applied to the cluster. This is the default.
+* `greedy_balanced_shards`: This mode uses a heuristic approach to find leadership movements that better balance leaders per shard. It applies any beneficial movements it finds.
 
-NOTE: In addition to the periodic heartbeat, leadership balancing can also occur when a xref:upgrade:rolling-upgrade.adoc#impact-of-broker-restarts[broker restarts] or when the controller leader changes (such as when a controller partition changes leader). The _controller leader_ manages the entire cluster. For example, when a broker is decommissioned, the controller leader creates a reallocation plan for all partition replicas allocated to that broker. The _partition leader_ then handles the reconfiguration for its Raft group.
+NOTE: In addition to the periodic heartbeat, leadership balancing can also occur when a xref:upgrade:rolling-upgrade.adoc#impact-of-broker-restarts[broker restarts] or when the controller leader changes (such as when a controller partition changes leader). The _controller leader_ manages the entire cluster. For example, when decommissioning a broker, the controller leader creates a reallocation plan for all partition replicas allocated to that broker. The _partition leader_ then handles the reconfiguration for its Raft group.
```

Comment:
why not just have _controller leader_ be a glossary link instead of italics here?